docs_user/assemblies/assembly_preparing-the-block-storage-service-for-adoption.adoc

```diff
@@ -4,9 +4,10 @@
 
 = Preparing the {block_storage} configurations for adoption
 
-As described in xref:planning-the-new-deployment_planning[Planning the new deployment], {block_storage_first_ref} is configured using
+The {block_storage_first_ref} is configured using
 configuration snippets instead of using configuration parameters
-defined by the installer.
+defined by the installer. For more information, see xref:planning-the-new-deployment_planning[Planning the new deployment].
```

Comment:
I notice in the xrefs, you have removed the context variable. Is that because the target resides in another assembly?

Reply:
I used "{context}" when the target was in the same assembly. If the target was in a different assembly, I used the actual context variable, e.g., "planning" or "migrating databases", etc.